tests/test_pacing.py

```diff
@@ -51,15 +51,15 @@ def do_not_log_me_man(self) -> None:
     @beat("Foobar...")
     def does_return_something(self, toggle: bool = False) -> Optional[int]:
         if toggle:
-            return 1
+            return 1  # pragma: no cover
```

Comment:
can we just... have `coverage` not report on the test files? This seems very silly to me.

Can we limit `coverage` to only look at `screenpy`?

Reply:
TBH, I wasn't sure if that would cause issues.  As long as it doesn't have any negative impact on measuring coverage of the package, I think that would be the smarter option.

Reply:
I also have no idea how to get coverage to omit the test modules when running coverage during execution of the tests.

Reply:
OK, so can we remove the `pragma` comments from the test files? It just seems bizarre and out of place to me. Or are you unable to tell Pycharm to ignore those files?

Reply:
Oh!  Yeah.  Of course.  I'll take them out.